test/unit/utils.py

```diff
@@ -36,3 +36,10 @@ def check_features(self, filename, features, opts=[]):
         self.assertEqual(p.stderr, '')
         self.assertEqual(p.stdout.split('\n')[:-1],
                          ['--enable-' + f for f in features])
+
+    # similar to assertEqual, but while ignoring line ending differences such
+    # as those between windows and unix
+    def assert_equal_ignoring_line_endings(self, left, right):
```

Comment:
We don't need to replace other usages of `assert_equal` with this?

Reply:
I think we would need to audit each one to see if line endings should be ignored there or not. That may explain some of the remaining failures in `unit`, but not the first few at least (which is when I gave up on trying to fix them all).